.github/CODEOWNERS

```diff
@@ -4,10 +4,11 @@
 *
 
 # team-specific repo segments
-/.github  @determined-ai/infrastructure
-/master   @determined-ai/backend
-/agent    @determined-ai/backend
-/proto    @determined-ai/backend
+/.github        @determined-ai/infrastructure
+/master         @determined-ai/backend
+/agent          @determined-ai/backend
+/proto          @determined-ai/backend
+/performance    @determined-ai/web
```

Comment:
maybe group entries by team?

Reply:
Aren't they already grouped?  Or do you mean put all the directories on one comma separated line?